src/components/MainPage.js

```diff
@@ -1,6 +1,10 @@
 // Cleaned up the bottom to remove excess code.  Added comments for methods  -Radium Branch
+/*
+	Added smooth scrolling for when the graph populates.
+ */
```

Comment:
I added smooth scrolling to this component to move the user to the `Graph` component when it populated the graph.